apps/omg_watcher/lib/omg_watcher.ex

```diff
@@ -14,8 +14,7 @@
 
 defmodule OMG.Watcher do
   @moduledoc """
-  Watcher is responsible for syncing and validating the child chain, and providing a secure interface to it.
-
-  For details see [here](README.md) and [here](docs/tesuji_blockchain_design.md)
+  Watcher is responsible for syncing and validating the child chain,
```

Comment:
note that the line limit length is set to 120 chars.

Also why remove the reference to the docs? They provide reference for what this module & app are responsible for.